Source/Core/Core/HW/EXI/EXI_DeviceIPL.h

```diff
@@ -31,6 +31,9 @@ class CEXIIPL : public IEXIDevice
 
   static void Descrambler(u8* data, u32 size);
 
+  static std::string FindIPLDump(const std::string& path_prefix);
```

Comment:
Why is FindIPLDump made public? I can't see any code that calls it from outside of EXI_DeviceIPL.cpp.